docs/validators/sigstore_cosign.md

```diff
@@ -223,6 +233,86 @@ Such environment variables can be injected into Connaisseur via `deployment.envs
     VAULT_TOKEN: secrettoken
 ```
 
+### Multi-signature verification
+
+> :warning: This is currently an experimental feature that might unstable over time.
+> As such, it is not part of our semantic versioning guarantees and we take the liberty to adjust or remove it with any version at any time without incrementing MAJOR or MINOR.
+> Central consideration for the experimental state is a bug in Python's [asyncio module](https://bugs.python.org/issue46364) by which multi-signature validation cannot be carried out concurrently in a secure manner.
+> As a result the current implementation uses sequential execution by which validation time scales linearly with the number of trust roots in scope.
+
+Connaisseur can verify multiple signatures for a single image.
+It is possible to configure a threshold number and specific set of required valid signatures.
+This allows to implement several advanced use cases (and policies):
+
+* Five maintainers of a repository are able to sign a single derived image, however at least 3 signatures are required for the image to be valid.
+* In a CI pipeline, a container image is signed directly after pushing by the build job and at a later time upon passing quality gates such as security scanners or integration tests, each with their own key (trust root). Validation requires all of these signatures for deployment to enforce integrity and quality gates.
+* A mixture of the above use cases whereby several specific trust roots are enforced (e.g. automation tools) and the overall number of signatures has to surpass a certain threshold (e.g. at least one of the testers admits).
+* Key rotation is possible by adding a new key as an additional key and require at least one valid signature.
```

Comment:
Nice! Wouldn't have thought of that use case. That's a cool list!

Reply:
thanks :slightly_smiling_face: